### DISCUSSION TITLE
Connecting to kernel: Python 3.9.7... loading to infinity

### DISCUSSION THREAD

**Discussion by leskovecg** [microsoft/vscode-jupyter#12526] [Questions and Answers]
When I start Interactive window loading goes to infinity. How can I solve this issue? 

![napaka](https://user-images.githubusercontent.com/62363084/211669311-21b6d7d6-23c9-49fd-8eda-6c6a88ef19cd.png)

**Comment by kvten** [comment]
Had similar issues in the past: #10413

Closed folder + VS Code and issue temporarily went away. But should've been fixed in one of the many releases since then.

Are you able to run python directly from the terminal?
